### PR TITLE
website: avoid confusion

### PR DESCRIPTION
**Pull request by pyr**
We are now taking liberties with the Cloudstack API, this
adapts our wording to reflect it.

### CODE REVIEW

**Comment by pyr** [comment]
The pointer to the API tooling can come back once its name changes.

File: website/content/_index.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-description: the Go library for Exoscale / CloudStack
+description: the Go library for Exoscale
 ---
 
 <a href="https://gopherize.me/gopher/9c1bc7cfe1d84cf43e477dbfc4aa86332065f1fd"><img src="gopher.png" align="right" alt=""></a>
 
 [![Build Status](https://travis-ci.org/exoscale/egoscale.svg?branch=master)](https://travis-ci.org/exoscale/egoscale) [![Maintainability](https://api.codeclimate.com/v1/badges/fcab3b624b7d3ca96a9d/maintainability)](https://codeclimate.com/github/exoscale/egoscale/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/fcab3b624b7d3ca96a9d/test_coverage)](https://codeclimate.com/github/exoscale/egoscale/test_coverage) [![GoDoc](https://godoc.org/github.com/exoscale/egoscale?status.svg)](https://godoc.org/github.com/exoscale/egoscale) [![Go Report Card](https://goreportcard.com/badge/github.com/exoscale/egoscale)](https://goreportcard.com/report/github.com/exoscale/egoscale)
 
-An API wrapper for the CloudStack based [Exoscale public cloud](https://www.exoscale.com).
+An wrapper for the [Exoscale public cloud](https://www.exoscale.com) API.
```
Comment (brutasse) [inline website/content/_index.md]: `A`

**Comment by greut** [comment]
we have links to CS everywhere.

- https://godoc.org/github.com/exoscale/egoscale
- also #127

File: website/layouts/index.html [hunk 2]
```diff
@@ -55,22 +55,7 @@ <h2 class="title">exo</h2>
...
-                </p>
-            </div>
-            <footer class="card-footer">
-                <p class="card-footer-item">
-                    <a href="cs">help</a>
-                </p>
-            </footer>
-        </div>
-    </div>
+
```
Comment (greut) [inline website/layouts/index.html]: and `config.toml` for the menu.

**Comment by pyr** [comment]
@greut good point, we'll need those taken care of as well. Since this is accretion, I'd rather get things right on first try.

**Pull request merged**